Userland/Libraries/LibSoftGPU/Config.h

```diff
@@ -14,9 +14,9 @@
 
 namespace SoftGPU {
 
-static constexpr bool ENABLE_STATISTICS_OVERLAY = false;
+static constexpr bool ENABLE_STATISTICS_OVERLAY = true;
```

Comment:
Did you mistakenly include these changes in this PR?

Reply:
Ugh. I did it again.